OrbitCore/CoreApp.h

```diff
@@ -45,11 +51,23 @@ class CoreApp {
                          const std::string& /*a_Module*/,
                          const std::string& /*a_Name*/) {}
   virtual void AddKeyAndString(uint64_t /*key*/, std::string_view /*str*/) {}
+  virtual void OnRemoteModuleDebugInfo(const std::vector<ModuleDebugInfo>&) {}
+  virtual void ApplySession(std::shared_ptr<Session> /*session*/) {};
   virtual const std::unordered_map<DWORD64, std::shared_ptr<class Rule> >*
   GetRules() {
     return nullptr;
   }
   virtual void RefreshCaptureView() {}
+
+  // Managers
+  orbit::TransactionManager* GetTransactionManager() {
```

Comment:
Do we want to have top level orbit namespace for all orbit code?

Reply:
We don't, I think we should add it.  Since this was a new class, I added it locally.  Maybe we should have a "using namespace orbit" in a core header during the transition to avoid having to use "orbit::" everywhere?